workers/unity/Packages/com.improbable.gdk.mobile/Worker/MobileWorkerConnector.cs

```diff
@@ -34,17 +39,72 @@ protected override ReceptionistConfig GetReceptionistConfig(string workerType)
 
         protected override LocatorConfig GetLocatorConfig()
         {
-            throw new System.NotImplementedException("The locator flow is currently not available for mobile workers.");
+            throw new NotImplementedException("The locator flow is currently not available for mobile workers.");
         }
 
-        protected override AlphaLocatorConfig GetAlphaLocatorConfig()
+        protected override AlphaLocatorConfig GetAlphaLocatorConfig(string workerType)
         {
-            throw new System.NotImplementedException("The alpha locator flow is currently not available for mobile workers.");
+            var pit = DevelopmentAuthentication.CreateDevelopmentPlayerIdentityTokenAsync(
+                RuntimeConfigDefaults.LocatorHost,
+                444,
+                new PlayerIdentityTokenRequest
+                {
+                    DevelopmentAuthenticationTokenId = DevelopmentAuthToken,
+                    PlayerId = $"Player-{Guid.NewGuid()}",
+                    DisplayName = "",
+                }
+            ).Get().Value;
+
+            if (!string.IsNullOrEmpty(pit.Error))
+            {
+                throw new ConnectionFailedException(pit.Error, ConnectionErrorReason.InvalidConfig);
```

Comment:
Is `InvalidConfig` actually a valid reason for failure in every case, here and below?
What if it's a different reason, for example if the service is down?